contents/product-engineers/activation-metrics.mdx

```diff
@@ -0,0 +1,286 @@
+---
+date: 2024-11-13
+title: How we found our activation metric (and how you can too)
+author:
+  - raquel-m-smith
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/hog_ql.png
+featuredImageType: full
+category: Engineering
+tags:
+  - Product analytics
+  - Product engineers
+  - Growth
+  - Inside PostHog
+crosspost:
+  - Blog
+---
+
+People are signing up for your product. Great! But before you start planning your IPO, you need to make sure they actually stick around. This is the goal of **activation**.
+
+The trouble with activation is that it is hard to define. Many say it is when a user reaches the "aha" moment of your product, but this is vague and different for everyone.
+
+For example, we have 8+ products and each has a different set of activation metrics from launching a survey to inviting a teammate to creating two or more dashboards. What made finding activation metrics tricky was that each product had 10+ events that **could** be part of its activation metric.
+
+This means we spent a lot of time discovering our products' activation metrics. We're sharing our process here to help you do the same.
+
+But before we jump in... what is activation _really_?
+
+## What is an activation metric?
+
+Activation metrics are a set of actions a new user performs that are correlated with greater retention.
+
+Focusing on retention solves the trickiest part of defining an activation metric: it could be _anything_. Does a pageview count as activation? How about inviting a teammate? Creating their first object? Depending on your product, any or all of these could be an activation metric. It is **retention** that makes it clear.
+
+At PostHog, we define activation metrics as a set of actions where a company is much more likely to be using the product 3 months after starting. For [session replay](/session-replay), after analyzing multiple potential metrics, we found that watching 5 or more replays and setting a replay filter at least once were the actions that resulted in the highest retention rate.
```

Comment:
> At PostHog, we define activation metrics as a set of actions where a company is much more likely to be using the product 3 months after starting.

This feels like we're redefining it again, even though we defned it above. I dont htink we need to mention the 3 or more months bit, since we already covered retention above.
```suggestion
Here are example of how've defined activation at PostHog:

- For [session replay](/session-replay) we found that watching 5 or more replays and setting a replay filter at least once were the actions that resulted in the highest retention rate.
- Another example...
- Another example...
```

Reply:
I see it as giving a specific example. This is just a narrower version of the definition above that works for us at PostHog.

Reply:
At the very least we should rephrase the opening sentence. What I mean by its confusing is that we do the following:
1. Heading is "what is activation"
2. First paragraph defines what it is ("activation metris are a set ....")
3. But then we say "we define activation metrics"). Which makes it seem like we introduced a second definition. 

Also, the sentence `we define activation metrics as a set of actions where a company is much more likely to be using the product 3 months after starting.` sounds like the definition of retention to me? Its a bit confusing.

IMO I dont think we need all of this. We've already given the defintions, so thats why I think if we include examples it makes everything clearer.

e.g.
- For session replay, our metrics are xyz
- For product analytics, our metrics are abc
- For surveys, our metrics are mny